test/unit/image_data_test.rb

```diff
@@ -30,6 +28,18 @@ class ImageDataTest < ActiveSupport::TestCase
     assert image_data.valid?
   end
 
+  test "requires images larger than 960x640 to be cropped" do
+    image_data = build_example("960x960_jpeg.jpg")
+    assert_not image_data.valid?
+    assert_equal ["must be 960px wide and 640px tall, but is 960px wide and 960px tall"], image_data.errors[:file]
+  end
+
+  test "images cannot be cropped if one side is too small" do
+    image_data = build_example("300x1000_png.png")
+    assert_not image_data.valid?
+    assert_equal ["must be 960px wide and 640px tall, but is 300px wide and 1000px tall"], image_data.errors[:file]
+  end
```

Comment:
Similarly here, I think re-wording our error message for images which are too small will improve the specificity of this test. It's hard to tell from reading the two tests what sets one apart from the other. Again using the `.kind_of?` method but using `assert_not` would make it clearer that in this scenario the image can't be cropped.